lib/dqt/client.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Dqt
+  class Client
+    def initialize(
+      headers: Dqt.configuration.client.headers,
+      host: Dqt.configuration.client.host,
+      params: Dqt.configuration.client.params
+    )
+      self.headers = headers
+      self.host = host
+      self.params = params
+    end
+
+    def api
+      @api ||= Api.new(client: self)
+    end
+
+    def get(path: "/", params: {})
+      http_client = HTTPClient.new
+
+      headers = {
+        'Content-Type': "application/json",
+      }.merge(send(:headers))
```

Comment:
Uhhh, `send` feels dirty. Can we not use `self.headers` here? Ditto about all other `send`s in this file.